WalletWasabi.Fluent/ViewModels/Wallets/Send/TransactionPreviewViewModel.cs

```diff
@@ -30,7 +31,7 @@ public partial class TransactionPreviewViewModel : RoutableViewModel
 
 			AddressText = info.Address.ToString();
 
-			ConfirmationTimeText = "~20 minutes ";
+			ConfirmationTimeText = $"~{TextHelpers.TimeSpanToFriendlyString(info.ConfirmationTimeSpan)} ";
```

Comment:
perhaps this could work/flow better?
```suggestion
			ConfirmationTimeText = $"Approximately {TextHelpers.TimeSpanToFriendlyString(info.ConfirmationTimeSpan)}.";
```

Reply:
using `Approximately` ACK.
`.` at the end, I am not sure. There is no sentence closing anywhere